components/script/dom/webglrenderingcontext.rs

```diff
@@ -1039,6 +1039,147 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
     }
 
     // https://www.khronos.org/registry/webgl/specs/latest/1.0/#5.14.10
+    fn Uniform2fv(&self,
+                  _cx: *mut JSContext,
+                  uniform: Option<&WebGLUniformLocation>,
+                  data: Option<*mut JSObject>) {
+        let data = match data {
+            Some(data) => data,
+            None => return self.webgl_error(InvalidValue),
+        };
+
+        if let Some(data) = array_buffer_view_to_vec_checked::<f32>(data) {
+            if data.len() < 2 {
+                return self.webgl_error(InvalidOperation);
+            }
+
+            self.Uniform2f(uniform, data[0], data[1]);
```

Comment:
I don't see how this works. This will only work for `glUniform2fv(uniform, 1, data)`, i.e. `uniform vec2 foo` or `uniform vec2 foo[1]`, but not for `uniform vec2 foo[10]`, which is set as `glUniform2fv(uniform, 10, data)`. In this case I'm assuming the WebGL API internally figures out the size because hey, we're not in C++-land anymore, but the behavior should be the same. We'll need a new message for it.

@emilio does this sound correct?